integration/class-jetpack.php

```diff
@@ -104,9 +104,11 @@ public static function add_reader_link( $actions, $item ) {
 		return array_merge(
 			array(
 				'reader' => sprintf(
-					'<a href="%s" target="_blank">%s</a>',
+					'<a href="%1$s" target="_blank">%2$s<span class="screen-reader-text"> %3$s</span><span aria-hidden="true" class="dashicons dashicons-external"></span></a>',
```

Comment:
```suggestion
					'<a href="%1$s" target="_blank">%2$s<span class="screen-reader-text"> %3$s</span><span aria-hidden="true" class="dashicons dashicons-external" style="font-size:1em;line-height:1.7;"></span></a>',
```
Let's make it a bit easier on the eyes:

<img width="482" height="168" alt="Image" src="https://github.com/user-attachments/assets/3860c15d-66d6-43a2-84b6-e9c4d962759b" />

Reply:
https://github.com/Automattic/wordpress-activitypub/pull/2247#discussion_r2391007255

Reply:
Not sure what that's trying to say? My suggestion doesn't change the markup or affect a11y, as far as I'm aware.

If it's about not using custom CSS, in this case it serves a purpose.

Reply:
Consistency!?

<img width="555" height="250" alt="Screenshot 2025-09-30 at 15 59 55" src="https://github.com/user-attachments/assets/c77b0f84-30ec-47c4-badd-26d0ef0e5bab" />

I added some CSS. I prefer that over inline CSS.